brave/brave-core
# ledger::VisitData is now defined as mojom

Fixes brave/brave-browser#4693

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests && npm run test-security`) on

## Test Plan:

1. Start Brave and enable Rewards
2. Ensure all panel functionality is retained.


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
